translations/hu.json

```diff
@@ -0,0 +1,2983 @@
+      }
+    ],
+    "responses": {
+      "setPing": ["Beállítva a **{event}**-re, hogy pingelődjön a **{role}** role."],
+      "removedPing": ["Eltávolítva a pingelődő role a **{event}**-re."]
+    }
+  },
+  "skyblock mayor": {
+    "name": "mayor",
+    "description": "Kijelzi a jelenlegi SkyBlock mayor-t, és a jelenlegi választást is.",
+    "options": [],
+    "responses": {
+      "mayorTitle": ["Hypixel SkyBlock Mayor"],
+      "mayorDescription": ["A jelenlegi mayor **{mayor}**.\n\n{perks}"],
+      "electionName": ["Választás a {year:,} évre"],
+      "electionDescription": ["Összes szavazat: {votes:,}"]
+    }
+  },
+  "skyblock bingo": {
+    "name": "bingo",
+    "description": "Kijelzi a jelenlegi SkyBlock Bingo kártyát.",
+    "options": [],
+    "responses": {
+      "bingoTitle": ["Hypixel SkyBlock Bingo Card"],
+      "progress": [
+        "Haladás: {progress:,}\n",
+        "Követketző Szint: {needed:,}\n> {toNext:,} needed\n",
+        "Szint {tier}/{tiers}"
+      ],
+      "finishedProgress": [
+        "Haladás: {progress:,}\n",
+        "Szint {tier}/{tiers}\n",
+        "Cél teljesítve!"
+      ]
+    }
+  },
+  "player": {
+    "name": "játékos",
+    "description": "Szerezz információt egy játékosról a Hypixelen.",
+    "options": [
+      {
+        "name": "játékos",
+        "description": "A játékos akiről információt szerzünk."
+      }
+    ],
+    "responses": {
+      "neverJoined": [
+        "**{name}** sose lépett be a Hypixel-re."
+      ],
+      "description": [
+        "Felhasználónév: {name}\n",
+        "UUID: {uuid}\n",
+        "Discord ID: {userid}\n",
+        "Rang: {rank}\n",
+        "Guild: {guild}\n",
+        "Első bejelentkezés: {first_login}\n",
+        "Legutolsó bejelentkezés: {last_login}\n",
+        "Legutolsó kijelentkezés: {last_logout}\n",
+        "Hypixel XP: {network_xp:,.2f}\n",
+        "Hypixel szint: {network_level:,.0f}\n",
+        "Minecraft verzió: {version}\n",
+        "Karma: {karma}\n",
+        "Achievement pontok: {achievement_points}\n",
+        "Státusz: {status}"
+      ],
+      "title": [
+        "{name}'s Hypixel profilja"
+      ]
+    }
+  },
+  "locate": {
+    "name": "bemérés",
+    "description": "Bemér egy játékost a Hypixel szerveren.",
+    "options": [
+      {
+        "name": "játékos",
+        "description": "A bemérendő játékos."
+      }
+    ],
+    "responses": {
+      "neverJoined": [
+        "**{name}** sose csatlakozott a Hypixel-re."
+      ],
+      "title": ["{name}'s Státusza"],
+      "offline": ["**{name}** jelenleg offline."],
+      "noData": ["Nem voltam képes **{name}** státuszát lehozni."],
+      "onlineCantLocate": ["**{name}** jelenleg online, de a státuszuk kikapcsolt API beállítást jeleznek."],
+      "online": ["**{name}** jelenleg online. Viszont nem tudom, hogy hol van."],
+      "onlineWithMode": ["**{name}** jelenleg {game}-t játszik.\nMód: {mode}."],
```

Comment:
Slide this in here as well
https://github.com/SkyKings-Guild/translations/blob/52c3263324591342a900b9b8ace1f073fc2ee300/translations/en-US.json#L1531